### PR TITLE
Mlt 284 vcp iter

### PR DESCRIPTION
**Pull request by chloeahampton**
What does this PR do?
---------------------

Previously, users were seeing repeat tiles on VCPs, and also having trouble actually iterating over VCPs, because of the way that iterator method was handling elasticsearch results from VS. This PR corrects this issue by fetching tiles from individual datapoint IDs instead of via slices/"indexes" directly from VS. 

What steps, if any, do reviewers need to take to set up their environments to test this properly?
---------------------
 - pull n go

Acceptance criteria
---------------------

-

Known issues
---------------------
- none

:white_check_mark: Checklist
---------------------
- [x] No additional work is in process. This PR should be ready for merge right now.
- [ ] Tests are included
- [x] There is at least one assignee

### CODE REVIEW

File: pyveda/vedaset/veda/api.py [hunk 1]
```diff
@@ -402,17 +402,26 @@ def from_id(cls, _id):
         doc['properties']['id'] = _id
         return cls.from_doc(doc)
 
+    # def __iter__(self):
```
Comment (rjpolackwich) [inline pyveda/vedaset/veda/api.py]: Can this commented out stuff be removed?

File: pyveda/vedaset/store/vedabase.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 from pyveda.vedaset.base import BaseDataSet, BaseSampleArray
 from pyveda.vedaset.interface import SerializedVariableArray, PartitionedIndexArray, ArrayTransformPlugin
 from pyveda.frameworks.batch_generator import VedaStoreGenerator
-from pyveda.vv.labelizer import Labelizer
+from pyveda import vv
```
Comment (rjpolackwich) [inline pyveda/vedaset/store/vedabase.py]: why change the import line to import the whole vv module instead of Labelizer object?

Comment (chloeahampton) [reply pyveda/vedaset/store/vedabase.py]: I can't remember why I did this... Will look and see if its actually necessary

File: pyveda/vedaset/veda/api.py (hunk 1 above)
Comment (chloeahampton) [reply pyveda/vedaset/veda/api.py]: yes

**Comment by rjpolackwich** [comment]
For the sake of comprehensiveness and transparency into the issue, could you expand on the source of the problem that this PR fixes? Ie, why did we have repeating tiles in the first place, and why does this fix that?

File: pyveda/vedaset/store/vedabase.py [hunk 3]
```diff
@@ -89,17 +89,12 @@ def clean(self, count=None, include_background_tiles=True):
...
         mltype = self._vset.mltype
-        Labelizer(self, mltype, count, classes, include_background_tiles).clean()
+        vv.labelizer.Labelizer(self, mltype, count, classes, include_background_tiles).clean()
 
     def preview(self, count=10, include_background_tiles=True):
-        """
-        Page through VedaCollection data and flag bad data.
-        Params:
-            count: the number of tiles to clean
-        """
```
Comment (cderwin) [inline pyveda/vedaset/store/vedabase.py]: Why did you remove the docstring?

Comment (chloeahampton) [reply pyveda/vedaset/store/vedabase.py]: it's technically not correct, and I just happened to notice that while I was working on this PR. I'll go in and fix the docstrings in dev directly, as there are quite a few that are wrong

Comment (rjpolackwich) [reply pyveda/vedaset/store/vedabase.py]: Ticket MLT-290 addresses this docstring work
